build.gradle.kts

```diff
@@ -1,64 +0,0 @@
-plugins {
```

Comment:
Doing stuff in the root build.gradle.kts file like this is adding plugins to your root project (unnecessary) and also. the subprojects and allprojects blocks should not really be used anymore. These lead to all sorts of ordering and timing issues. 

The suggested approach these days is to use [convention plugins](https://docs.gradle.org/current/samples/sample_convention_plugins.html)